actnow/accounts/serializers.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth import get_user_model
+from django.utils.translation import gettext_lazy as _
+from rest_framework import serializers
+
+from .utils import email_address_exists, username_exists
+
+User = get_user_model()
+
+
+class RegisterSerializer(serializers.Serializer):
```

Comment:
```suggestion
class UserRegistrationSerializer(serializers.Serializer):
```

Don't we use nouns in RESTful APIs?

Reply:
Yeah. We do. Updating this.